lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -622,16 +622,20 @@ def process(self, forecast_probabilities, no_of_percentiles=None,
         return forecast_at_percentiles
 
 
-class FromMeanAndVariance():
+class ConvertLocationAndScaleParameters():
     """
     Base Class to support the plugins that compute percentiles and
-    probabilities from the mean and variance.
+    probabilities from the location and scale parameters.
     """
 
     def __init__(self, distribution="norm", shape_parameters=None):
         """
         Initialise the class.
 
+        .. Further information is available in:
```

Comment:
Not sure if this paragraph actually needs to be separated out into an extended_documentation or not

Reply:
Yes, I think this probably isn't necessary, so I've moved this paragraph into the docstring.